### PR TITLE
explicit way to express that docker.list are different choices and not just one big file

### PR DESCRIPTION
**Pull request by gissehel**
I ran in the same issue as described in #15692 

My problem was that when I followed the installation instructions, I went too fast and didn't realize that the docker.list file in the ubuntu instructions was 4 choices. I copied the whole content (as there was one big `code` block) thinking that `apt` would find the right line based on the current distribution I use.

I'm sure with just 4 `code` blocks I wouldn't have that problem.

Related documentation page: http://docs.docker.com/engine/installation/ubuntulinux/

Note: The pull request also contains change for debian as it's exactly the same problem.

### CODE REVIEW

File: docs/installation/debian.md [hunk 1]
```diff
@@ -63,12 +63,20 @@ from the new repository:
...
 
-         # Debian Wheezy
-        deb https://apt.dockerproject.org/repo debian-wheezy main
-        # Debian Jessie
-        deb https://apt.dockerproject.org/repo debian-jessie main
-        # Debian Stretch/Sid
-        deb https://apt.dockerproject.org/repo debian-stretch main
+     - On Debian Wheezy
+
+            # Debian Wheezy
```
Comment (thaJeztah) [inline docs/installation/debian.md]: nit: perhaps we should remove the comments inside the code block, because they're only repeating the line before

**Comment by thaJeztah** [comment]
I think this makes sense, we could debate wether this is the best approach (layout-wise), but I think it's clearer than the current version.

I added one suggestion, but LGTM

**Comment by LK4D4** [comment]
@gissehel Would you mind to fix @thaJeztah suggestion?
ping @moxiegirl @SvenDowideit

**Comment by moxiegirl** [comment]
LGTM @gissehel thank you for the contribution

**Comment by gissehel** [comment]
Ok, I've removed the lines, sorry for the delay.

**Comment by thaJeztah** [comment]
thanks @gissehel, LGTM!

**Pull request merged**